src/main/java/com/minecolonies/util/StructureWrapper.java

```diff
@@ -756,42 +661,42 @@ public Item getItem()
     }
 
     /**
-     * @return The name of the schematic.
+     * @return The name of the structure.
      */
     public String getName()
     {
         return name;
     }
 
     /**
-     * @return The height of the schematic.
+     * @return The height of the structure.
      */
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'block' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)